clustered/server/src/main/java/org/ehcache/clustered/server/ServerStoreImpl.java

```diff
@@ -90,4 +90,30 @@ public void clear() {
   public void close() {
     store.close();
   }
+
+  // stats
+
+  public long allocatedMemory() {return store.allocatedMemory();}
```

Comment:
@ljacomet :  yes Mathieu is right, I was able to make ServerStoreImpl implement MapInternals.  The only problem I encountered was that MapInternals.getReprobeLength() returns an int and OffHeapServerStore.reprobeLength() returns a long.